merge_checks/runner.py

```diff
@@ -78,6 +78,6 @@ def run():
 
     set_commit_status(
         **(asdict(commit) | asdict(status)),
-        state=str((State.SUCCESS if checks_passed else State.FAILURE).value),
+        state=(State.SUCCESS if checks_passed else State.FAILURE),
```

Comment:
I think the parens are not needed syntactically here.

Reply:
You're right! Fixed.